backend/stakepoold/log.go

```diff
@@ -21,12 +21,7 @@ type logWriter struct{}
 
 func (logWriter) Write(p []byte) (n int, err error) {
 	os.Stdout.Write(p)
-	// Normally this if isn't required, but the tests in this main package need
-	// to work with uninitialized log rotators.
```

Comment:
Is this comment not accurate in this project?

Reply:
not anymore.  i took the logging out of the API that the _tests's check.  Without it, the tests would panic.